bin/reindex-rds

```diff
@@ -1,11 +1,8 @@
 #!/usr/bin/env bash
 set -e
-. bin/ci-process-bootstrap
-
-. bin/ci-start-processes
 
 ./bin/console rds:reindex
 
 echo "Waiting for empty queues"
```

Comment:
perhaps stick this comment into `./bin/wait-for-empty-queues`

Reply:
Done. The bigger scare for me is that this script was prior to this PR being run on both CI and Production. In this instance, running this on production would spawn a watcher and a worker bypassing `systemd` and if the script fails there would be dangling processes left behind.
This change should resolve that issue, although it assumes that a worker and watcher will already running.